### PR TITLE
Bug 1170637 - Unit test tiles view

### PR DESCRIPTION
**Pull request by hfiguiere**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=683131e0cede9cff6fafafd450b2a6e4cb4399cf

**Comment by hfiguiere** [comment]
Requires some changes from another PR.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4049b78fe27acf33e70058bb98a8606a6e7bdf42

File: apps/music/test/unit/metadata/mock_album_art_cache.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
+/* exported MockAlbumArtCache */
+'use strict';
+
+
+var MockAlbumArtCache = {
+  getThumbnailURL: function() {
+    // Mock: return a fake promise.
+    return {
+      then: function() { }
```
Comment (jimporter) [inline apps/music/test/unit/metadata/mock_album_art_cache.js]: Can you use a real Promise here? That is, `return Promise.resolve();`. That'll probably be less error-prone in the future (I just got bitten by this with the `MockLazyLoader`, which I had to fork for the music app).

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=378d2c95bba9e4be788089321fba1db8f96882db

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=518d4dd95dbb437b2af9f3404e0a1f992bed6dc7

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6fff4cdc41b907720305f02b376804932bbcefe1

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=cd3521bcde539b7a484d526f4c5b22109312b4a3

**Pull request merged**